Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -15,26 +15,40 @@ class EateryTabBarController: UITabBarController {
 
     // MARK: View controllers
 
-    let eateriesSharedViewController = EateriesSharedViewController()
+    let eateriesViewController = CampusEateriesViewController()
     let lookAheadViewController = LookAheadViewController()
     let brbViewController = BRBViewController()
-
+    
+    let eateriesNavigationController: EateryNavigationController
+    
+    init() {
+        self.eateriesNavigationController = EateryNavigationController(rootViewController: eateriesViewController)
+        
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
     override func viewDidLoad() {
         delegate = self
-        eateriesSharedViewController.tabBarItem = UITabBarItem(
+        
+        eateriesNavigationController.delegate = self
+        eateriesNavigationController.tabBarItem = UITabBarItem(
             title: nil,
             image: UIImage(named: "eateryTabIcon.png"),
             tag: 0
         )
-
+        
```

Comment:
Nit: delete whitespace! Would recommend turning on the setting to automatically remove whitespace from newlines in Xcode

Reply:
I tried the second solution on this StackOverflow post (with 245 likes).  Let me know if you mean something different.

https://stackoverflow.com/questions/1390329/trim-trailing-spaces-in-xcode#:~:text=Just%20go%20to%20Find%20And,error%20will%20be%20gone%20!!